src/main/java/seedu/address/model/person/ClientStatus.java

```diff
@@ -35,25 +35,55 @@ public String toString() {
 
     public final Status value;
 
+    /**
+     * Constructs a {@code ClientStatus}.
+     * Only to be used when reading Json files and by other methods in the {@code ClientStatus} class.
+     *
+     * @param status The ordinal of the status enumeration.
+     */
     public ClientStatus(int status) {
         value = Status.values()[status];
     }
```

Comment:
The JavaDoc for the ClientStatus constructor is clear and concise. It's good that you've specified the intended use cases for this constructor, such as reading from JSON files and internal usage within the ClientStatus class. However, consider adding a note about the range of acceptable values for the status parameter to prevent potential ArrayIndexOutOfBoundsException errors.